### PR TITLE
Add section on core translations

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

File: pages/docs/recipes/set-up-translations-in-a-frontend-module.en-US.mdx [hunk 1]
```diff
@@ -23,3 +23,13 @@ const ActiveVisitsTable = () => {
   return <h4>{t("activeVisits", "Active Visits")}</h4>;
 };
 ```
+
+Many common translation strings are provided by the framework—these are called "core translations." These can be accessed with the [`getCoreTranslation`](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#getcoretranslation) function. To use this function, simply replace `t` with `getCoreTranslation`. A Typescript type error will be shown if the key is not valid. The `CoreTranslationKey` Typescript type provides the list of avialable translation keys; the list can also be seen in the [API docs](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#getcoretranslation). The list includes common strings used in many apps, such as "Cancel," "Confirm," "Female," and "Other." The translation JSON files themselves are [in openmrs-esm-core](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/framework/esm-translations/translations).
```
Comment (denniskigen) [inline pages/docs/recipes/set-up-translations-in-a-frontend-module.en-US.mdx:27]: The framework provides many common translation strings, referred to as core translations, that are used in many frontend modules. These core translations are defined in the [CoreTranslationKey](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-translations/src/translations.ts#L15) type. These include common strings used in many apps, such as `Cancel`, `Confirm`, `Female`, and `Other`. The corresponding translation JSON files themselves live in the [esm-translations](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/framework/esm-translations/translations) app.

To use these translations, call the [`getCoreTranslation`](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#getcoretranslation) function. This function takes two arguments: the first is the key of the translation, and the second is the default value to return if the key does not exist. A Typescript type error is shown if the provided key does not exist in `CoreTranslationKey`.

**Review by denniskigen — COMMENTED** [review]
Thanks, @brandones. I've suggested some improvements to try and make the content clearer.

File: pages/docs/recipes/set-up-translations-in-a-frontend-module.en-US.mdx (hunk 1 above)
Comment (brandones) [reply pages/docs/recipes/set-up-translations-in-a-frontend-module.en-US.mdx:27]: Some things about this are indeed better, but I feel like this "buries the lede." `getCoreTranslation` is the headline here.


> The framework provides a set of *core translations* that are used in many frontend modules. These can be used with the [`getCoreTranslation`](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#getcoretranslation) function. This function takes two arguments: the first is the key of the core translation, and the second is the default value to return if the requested translation is not found.  A Typescript type error is shown if the provided key is not valid.
> 
> The valid core translations are defined in the [CoreTranslationKey](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-translations/src/translations.ts#L15) type. These include common strings used in many apps, such as `Cancel`, `Confirm`, `Female`, and `Other`. The corresponding translation JSON files themselves live in the [esm-translations](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/framework/esm-translations/translations) app.

?

Comment (denniskigen) [reply pages/docs/recipes/set-up-translations-in-a-frontend-module.en-US.mdx:27]: Fair. I think I erred too much on the opposite side of brevity with those suggestions.

Comment (brandones) [reply pages/docs/recipes/set-up-translations-in-a-frontend-module.en-US.mdx:27]: I was doubting myself on this (I have a bad habit of not listening to editors as much as I ought to) so I ran it by some bots. ("Which of these is better documentation for the "core translations" feature of a Javascript framework? A or B?") ChatGPT agrees with me. Gemini on the other hand will just tell you the one that you put in second is better, whichever one that is :laughing:

**Review by denniskigen — COMMENTED** [review]
Thanks a bunch, @brandones!

**Review by denniskigen — APPROVED** [review]

**Pull request merged**